Web/Edubase.Web.UI/Areas/Establishments/Models/Validators/CreateEstablishmentViewModelValidator.cs

```diff
@@ -12,14 +12,14 @@ public CreateEstablishmentViewModelValidator(IEstablishmentReadService establish
         {
             RuleFor(x => x.Name).NotEmpty().WithMessage("Please enter an establishment name");
             RuleFor(x => x.LocalAuthorityId).NotEmpty().WithMessage("Please select a local authority");
-
```

Comment:
this validation has been removed, we want to ensure that the usual JS route continues in the same manner so just worth confirming this hasn't influenced that journey